pylot/operator_creator.py

```diff
@@ -567,27 +569,35 @@ def _add_lidar_driver(vehicle_id_stream, release_sensor_stream, lidar_setup):
 def add_imu(transform, vehicle_id_stream, name='imu'):
```

Comment:
```suggestion
def add_imu(transform: pylot.utils.Transform, vehicle_id_stream: erdos.Stream, name: str='imu') -> (erdos.Stream, pylot.drivers.sensor_setup.IMUSetup):
```
Please double check the types!